pkg/nsx/cluster.go

```diff
@@ -418,3 +425,83 @@ func (nsxVersion *NsxVersion) featureSupported(feature int) bool {
 	}
 	return false
 }
+
+func (cluster *Cluster) createHttpRequest(api string, ep *Endpoint) (*http.Request, error) {
+	return http.NewRequest("GET", fmt.Sprintf("%s://%s/%s", ep.Scheme(), ep.Host(), api), nil)
+}
+
+func (cluster *Cluster) getLicenseFromNsx() (*util.NsxLicense, error) {
+	ep := cluster.endpoints[0]
+	req, err := cluster.createHttpRequest("api/v1/licenses/licensed-features", ep)
```

Comment:
suggest to define the license URL as a const

Reply:
done